### PR TITLE
Update the lower-bound of the lablgtk dependency.

### LINKED ISSUE #6509: CoqIDE 8.7.1 for macOS crashes upon copy / pasting
#### Description of the problem

The CoqIDE crushes every time when I hit `cmd + C`. The IDE I install is from `https://github.com/coq/coq/releases/download/V8.7.1/coq-8.7.1-installer-macos.dmg`.

### PR DESCRIPTION
**Pull request by Zimmi48**
**Kind:** documentation / infrastructure.

Closes #6509.

We should also check whether the "lablgtk-extras" package is really needed.

### CODE REVIEW

**Comment by Zimmi48** [comment]
I've only updated the ocamlfind version test, but should we continue to support alternative ways of finding lablgtk?
Also, it seems that there is a bug with lablgtk 2.18.3 which makes `ocamlfind query -format "%v" lablgtk2` report 2.18.0 (whereas it works correctly for lablgtk 2.18.4). So let's make the lower bound 2.18.4, no?

**Comment by Zimmi48** [comment]
Updated to require lablgtk 2.18.4.
In case, the `-lablgtkdir` option is used, no version check is done and a warning is always printed. Feel free to propose a refinement to this approach.

File: INSTALL.ide [hunk 1]
```diff
@@ -39,7 +39,7 @@ COMPILATION REQUIREMENTS
   install GTK+ 2.x, should you need to force it for one reason
   or another.)
 - The OCaml bindings for GTK+ 2.x, lablgtk2 with support for gtksourceview2.
-  You need at least version 2.16.
+  You need at least version 2.18.4.
```
Comment (ejgallego) [inline INSTALL.ide]: Please, revert back to 2.18.4, as this is the lablgtk version available in Debian stable as of today.

**Review by ejgallego — CHANGES_REQUESTED** [review]

File: INSTALL.ide (hunk 1 above)
Comment (Zimmi48) [reply INSTALL.ide]: OK, thanks for this valuable info. What do you think of the version test. Should it be based on a heuristic (availability of a given function)? Any other idea?

Comment (ejgallego) [reply INSTALL.ide]: That's a problem I don't know how to solve, maybe emit a warning if we find 2.18.0?

Comment (ejgallego) [reply INSTALL.ide]: So 2.18.0 = OK + warning, >= 2.18.3 OK, everything else, fail.

Comment (Zimmi48) [reply INSTALL.ide]: Sounds good.

Comment (Zimmi48) [reply INSTALL.ide]: Fixed.

Comment (ejgallego) [reply INSTALL.ide]: Thanks Théo !!

**Review by ejgallego — APPROVED** [review]
Looks good to me, thanks @Zimmi48 for the hard work.

**Pull request merged**